prototyping/17_flag_anomalies.R

```diff
@@ -0,0 +1,51 @@
+rm(list = ls())
+
+library(tidyverse)
+library(behindbarstools)
+
+# ------------------------------------------------------------------------------
+
+# Only flag when cumulative counts drop by DIFF_THRESHOLD
+DIFF_THRESHOLD <- 5 
+
+# AND that decline is more than PCT_THRESHOLD percent 
+PCT_THRESHOLD <- 0.1
+
+# ------------------------------------------------------------------------------
+
+GITHUB_PATH <- "https://raw.githubusercontent.com/uclalawcovid19behindbars/"
+FAC_PATH <- "data/master/historical-data/historical_facility_counts.csv"
+STATE_PATH <- "data/master/historical-data/historical_state_counts.csv"
```

Comment:
Using the same versions of the files Hyperobjekt will be to power the dashboard — NOT the latest and greatest from the server. So if you make a change to the server, it won't show up until after the next time we scrape/write data again.